setup_env.sh

```diff
@@ -0,0 +1,70 @@
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+if [ "$1" == "-h" ]; then
+    echo "Argument -c allows to install git global name and email."
+    echo "Example setup_env.sh -c username usermail"
+    exit 0
+fi
+
+cp /etc/apt/sources.list /etc/apt/sources.list.backup
+
+. /etc/lsb-release
+
+packages=(
+libssl-dev
+libudev-dev
+libavahi-client-dev
+sqlite3
+libsqlite3-dev
+automake1.11
+libexpat1-dev
+ccache
+libxml2
+libxml2-dev
+git
+libpulse-dev
+subversion
+doxygen
+graphviz
+mscgen
+libbluetooth3
+libbluetooth-dev
+bluez-tools
+chromium-browser
+libgl1-mesa-dev
+libgtest-dev
+binutils
+)
+
+if [ $DISTRIB_RELEASE!="14.04" ]; then
```

Comment:
@AByzhynar Ubuntu 15 and up does not have cmake 2.8 in standard packages. This instalation way was tested with Ubuntu 15.10 and 16.04.